lineage/tHMM.py

```diff
@@ -106,7 +106,7 @@ def get_BIC(self, LL: float, num_cells: int, atonce=False) -> Tuple[float, float
             degrees_of_freedom += self.num_states * (self.num_states - 1)
 
         if atonce:  # assuming we are fitting all 4 concentrations at once and we have cell cycle phase specific distributions.
-            degrees_of_freedom += self.num_states * 4.5
+            degrees_of_freedom += self.num_states * 4.25
```

Comment:
Where does this change come from?

Reply:
That should have been 4.25.. which shows 4 scales and 1/4 shape -- since we assume all conditions have a shared shape. 
Sorry, I don't know when I had made it 4.5, even in the methods that we explained DOF we have said it should be 4.25.
It should hold true now that we have those 4 conditions together.

Reply:
Huh.. actually, that also might be wrong.. I can't remember why Bernoulli parameter is not a part of it

Reply:
Apologies.. I looked again and it was my mistake.. I will fix it.